source/lex.tex

```diff
@@ -288,6 +288,25 @@
 The \grammarterm{universal-character-name} construct provides a way to name
 other characters.
 
+\begin{bnf}
+\nontermdef{n-char}: one of\br
+    A B C D E F G H I J K L M N O P Q R S T U V W X Y Z\br
+    0 1 2 3 4 5 6 7 8 9\br
+    \unicode{002d}{hyphen-minus}\br
+    \unicode{0020}{space}\br
```

Comment:
It seems the digits here are italics.
Please wrap in `\textnormal` for now (the macro ought to take care of that).